pkg/api/customization/app/app.go

```diff
@@ -169,6 +178,12 @@ func (w Wrapper) ActionHandler(actionName string, action *types.Action, apiConte
 		if err := access.ByID(apiContext, &projectschema.Version, projectv3.AppRevisionType, revisionID, &appRevision); err != nil {
 			return err
 		}
+
+		err := w.validateRancherVersion(appRevision.Status.ExternalID)
```

Comment:
Say if I am on rancher:v2.1.2 and the min version for an app (v0.2) is specified as v2.1.2. Now if I did a rancher downgrade to v:2.1.1. Will I be able to rollback the app to v0.1 whose min version was v2.1.1?

Reply:
Yeah, assuming the check passes but if the min matches the current version it will work.